### PR TITLE
added azure connection string

### PR DESCRIPTION
**Pull request by bharel**

### CODE REVIEW

File: yellowbox/extras/azure_storage.py [hunk 1]
```diff
@@ -43,6 +46,15 @@ def stop(self, signal: Union[str, int] = 'SIGKILL'):
...
         return get_ports(self.container)[BLOB_STORAGE_DEFAULT_PORT]
 
+    @property
+    def connection_string(self):
+        return textwrap.dedent(f"""
+        DefaultEndpointsProtocol = http;
+        AccountName = {self.account_name};
+        AccountKey = {self.account_key};
+        BlobEndpoint = http://localhost:{self.client_port()}/{self.account_name};
+        """).replace("\n", "")
```
Comment (bentheiii) [inline yellowbox/extras/azure_storage.py:56]: don't do replace and dedent, do:
```
return "a"\
    "b"\
    "c"\
    ...
```

File: yellowbox/extras/azure_storage.py [hunk 2]
```diff
@@ -33,6 +34,8 @@ def __init__(self, docker_client: DockerClient, image: str = "mcr.microsoft.com/
         container = create_and_pull(
             docker_client, image, "azurite-blob --blobHost 0.0.0.0", publish_all_ports=True)
         super().__init__(container, **kwargs)
+        self.account_name = DEFAULT_ACCOUNT_NAME
+        self.account_key = DEFAULT_ACCOUNT_KEY
```
Comment (bentheiii) [inline yellowbox/extras/azure_storage.py:38]: add parameters to __init__? if these attributes are needed before `start`, `run` will be unusable

File: yellowbox/extras/azure_storage.py [hunk 3]
```diff
@@ -43,6 +46,15 @@ def stop(self, signal: Union[str, int] = 'SIGKILL'):
     def client_port(self):
         return get_ports(self.container)[BLOB_STORAGE_DEFAULT_PORT]
 
+    @property
+    def connection_string(self):
+        return textwrap.dedent(f"""
```
Comment (bentheiii) [inline yellowbox/extras/azure_storage.py:51]: Mom! Bar didn't document his code

**Review by bentheiii — CHANGES_REQUESTED** [review]
add a test to check that this works

File: yellowbox/extras/azure_storage.py (hunk 2 above)
Comment (bharel) [reply yellowbox/extras/azure_storage.py:38]: It's not a parameter. It's attributes. Is it better now?

**Review by bentheiii — APPROVED** [review]

**Pull request merged**